### PR TITLE
Correct spelling & grammar in 4.4 components/

### PR DESCRIPTION
**Pull request by gnito-org**

### CODE REVIEW

File: components/inflector.rst [hunk 1]
```diff
@@ -60,5 +60,5 @@ forms::
     Inflector::singularize('indices'); // ['index', 'indix', 'indice']
     Inflector::singularize('leaves');  // ['leaf', 'leave', 'leaff']
 
-    Inflector::pluralize('matrix');    // ['matricies', 'matrixes']
+    Inflector::pluralize('matrix');    // ['matrices', 'matrixes']
```
Comment (OskarStark) [inline components/inflector.rst:63]: If this is correct we should check our code of the pluralize method too, maybe we also did this mistake there 😉

File: components/lock.rst [hunk 2]
```diff
@@ -51,7 +51,7 @@ method will try to acquire the lock::
 
     if ($lock->acquire()) {
         // The resource "pdf-invoice-generation" is locked.
-        // You can compute and generate invoice safely here.
+        // You can compute and generate invoices safely here.
```
Comment (OskarStark) [inline components/lock.rst]:
```suggestion
        // You can compute and generate the invoice safely here.
```

Cannot view the context of the code example, I am currently on a phone

File: components/options_resolver.rst [hunk 3]
```diff
@@ -434,8 +434,8 @@ if you need to use other options during normalization::
         }
     }
 
-To normalize a new allowed value in sub-classes that are being normalized
-in parent classes use :method:`Symfony\\Component\\OptionsResolver\\OptionsResolver::addNormalizer`.
+To normalize a new allowed value in subclasses that are being normalized
+in parent classes, use :method:`Symfony\\Component\\OptionsResolver\\OptionsResolver::addNormalizer`.
```
Comment (OskarStark) [inline components/options_resolver.rst]:
```suggestion
in parent classes, use :method:`Symfony\\Component\\OptionsResolver\\OptionsResolver::addNormalizer` method.
```
?

File: components/inflector.rst (hunk 1 above)
Comment (gnito-org) [reply components/inflector.rst:63]: https://github.com/symfony/symfony/pull/44632

**Pull request merged**